service/dap/server.go

```diff
@@ -379,7 +379,7 @@ func (s *Server) Stop() {
 
 	if s.listener != nil {
 		// If run goroutine is blocked on accept, this will unblock it.
-		_ = s.listener.Close()
```

Comment:
Same as the other PR.

Reply:
This is actually removing the `_ = ` not adding it.